typeagent/aitools/embeddings.py

```diff
@@ -37,6 +47,7 @@ class AsyncEmbeddingModel:
     async_client: AsyncOpenAI | None
     azure_endpoint: str
     azure_api_version: str
+    encoding_name: str | None
```

Comment:
Looks like you don't ever set this variable. Instead, you set several others, below on lines 114-120, that you might want to declare here.